### PR TITLE
feat(ipfs): documentation about ipfs project

### PR DESCRIPTION
**Pull request by OnsagerHe**
### Your checklist for this pull request

- [X] Check that the commit messages match our requested structure.
- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
The purpose of this PR is to add the IPFS project to the documentation. It contains the necessary information about the Scaleway CLI. There is also a quickstarter and a short explanation of the concepts.

### CODE REVIEW

File: storage/ipfs/api-cli/ipfs-cli.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+This operation allows you to add Scaleway remote service in your CLI IPFS.
+
+**CLI command**
+```
+ipfs pin remote service add scaleway https://fr-par.stg.pine.labs.scw.cloud/<your-volume-id>/ <SECRET_KEY>
+```
+
+<Message type="note">
+    If no service exists with the name, there is no response expected
```
Comment (ofranc) [inline storage/ipfs/api-cli/ipfs-cli.mdx:40]:
```suggestion
    If no service exists with the name, there is no response expected.
```

File: storage/ipfs/api-cli/ipfs-cli.mdx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+**Sample response**
+```
+CID:    QmaGNR5PedL8kiTUMtsPeFFodL9HnYBMuUqAEQrv6CQH4F
+Name:
+Status: pinned
+```
+
+<Message type="note">
+    You can have four status: 'queued', 'pinning', 'pinned' or 'failed'
```
Comment (ofranc) [inline storage/ipfs/api-cli/ipfs-cli.mdx:59]:
```suggestion
    You can have four status: 'queued', 'pinning', 'pinned' or 'failed'.
```

File: storage/ipfs/api-cli/ipfs-operations.mdx [hunk 3]
```diff
@@ -0,0 +1,178 @@
+---
+meta:
+  title: Understanding IPFS Storage ipfs operations
```
Comment (ofranc) [inline storage/ipfs/api-cli/ipfs-operations.mdx:3]: C'est normal le "ipfs operations" ici ? same pour les lignes du bas

File: storage/ipfs/api-cli/ipfs-operations.mdx [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+tags: ipfs storage ipfs-storage volume
+dates:
+  validation: 2023-04-23
+  posted: 2023-04-23
+categories:
+  - storage
+  - ipfs-storage
+---
+
+Scaleway IPFS allows you to store content on the IPFS network via CID or URL. You can [create](/storage/ipfs/api-cli/volumes-operations) and manage your volume  via the [Scaleway Command Line Interface](https://github.com/scaleway/scaleway-cli).
```
Comment (ofranc) [inline storage/ipfs/api-cli/ipfs-operations.mdx:17]:
```suggestion
Scaleway IPFS allows you to store content on the IPFS network via CID (Content-ID) or URL. You can [create](/storage/ipfs/api-cli/volumes-operations) and manage your volume via the [Scaleway Command Line Interface](https://github.com/scaleway/scaleway-cli).
```

File: storage/ipfs/api-cli/ipfs-operations.mdx [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+    - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/)
+    - You have generated your [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+    - You have downloaded and installed the [Scaleway CLI v2](https://github.com/scaleway/scaleway-cli#scaleway-cli-v2)
+</Message>
+
+## AddCID
```
Comment (ofranc) [inline storage/ipfs/api-cli/ipfs-operations.mdx:31]:
```suggestion
## Adding CID
```

File: storage/ipfs/api-cli/ipfs-operations.mdx [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+```
+PinID           22222222-2222-2222-2222-222222222222
+Status          queued
+CreatedAt       now
+Cid.Cid         Qmdi7ERksspfxWXfU8ATRUt7iCjZJbEbDrUoMDtjnbdTwo
+Cid.Name        newcid
+Cid.Meta.AppID  00000000-0000-0000-0000-000000000000
+```
+
+## AddURL
```
Comment (ofranc) [inline storage/ipfs/api-cli/ipfs-operations.mdx:60]:
```suggestion
## Adding URL
```

File: storage/ipfs/api-cli/ipfs-operations.mdx [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+
+This operation allows you to add content by CID in volume.
```
Comment (ofranc) [inline storage/ipfs/api-cli/ipfs-operations.mdx:62]:
```suggestion
This operation allows you to add content by URL in a volume.
```

File: storage/ipfs/api-cli/ipfs-operations.mdx [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+
+<Message type="requirement">
+    - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/)
+    - You have generated your [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+    - You have downloaded and installed the [Scaleway CLI v2](https://github.com/scaleway/scaleway-cli#scaleway-cli-v2)
+</Message>
+
+## AddCID
+
+This operation allows you to add content by CID in volume.
```
Comment (ofranc) [inline storage/ipfs/api-cli/ipfs-operations.mdx:33]:
```suggestion
This operation allows you to add content by CID in a volume.
```

File: storage/ipfs/api-cli/ipfs-operations.mdx [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+Cid.Cid         -
+Cid.Name        newcid
+Cid.Meta.AppID  00000000-0000-0000-0000-000000000000
+```
+
+<Message type="note">
+  If the response is ingest you can retrieve the CID of your url.
+</Message>
+
+## GetPinID
```
Comment (ofranc) [inline storage/ipfs/api-cli/ipfs-operations.mdx:93]:
```suggestion
## Getting Pin ID
```

File: storage/ipfs/api-cli/ipfs-operations.mdx [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+Cid.Name        value:"newcid"
+Cid.Meta.AppID  00000000-0000-0000-0000-000000000000
+```
+
+<Message type="note">
+  If the response is ingest you can retrieve the CID of your url.
+</Message>
+
+<Message type="important">
+If status is failed, you can verify information about CID or URL and retry.
```
Comment (ofranc) [inline storage/ipfs/api-cli/ipfs-operations.mdx:128]:
```suggestion
If the status is failed, you can verify information about the CID or the URL and retry.
```

File: storage/ipfs/api-cli/volumes-operations.mdx [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+ID         11111111-1111-1111-1111-111111111111
+ProjectID  00000000-0000-0000-0000-000000000000
+Region     fr-par
+CountPin   0
+CreatedAt  now
+UpdatedAt  now
+Name       demo
+```
+
+## DeleteVolume
```
Comment (ofranc) [inline storage/ipfs/api-cli/volumes-operations.mdx:118]:
```suggestion
## Deleting a Volume
```

File: storage/ipfs/api-cli/volumes-operations.mdx [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+ProjectID  00000000-0000-0000-0000-000000000000
+Region     fr-par
+CountPin   0
+CreatedAt  21 seconds ago
+UpdatedAt  21 seconds ago
+Name       demo
+```
+
+## ListVolume
+This operation allows you to list all volumes in project id mentioned in the command.
```
Comment (ofranc) [inline storage/ipfs/api-cli/volumes-operations.mdx:173]:
```suggestion
This operation allows you to list all volumes in the project ID mentioned in the command.
```

File: storage/ipfs/api-cli/ipfs-cli.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (OnsagerHe) [reply storage/ipfs/api-cli/ipfs-cli.mdx]: I rephrased, but that was if the service name didn't already exist.

File: storage/ipfs/api-cli/ipfs-operations.mdx (hunk 3 above)
Comment (OnsagerHe) [reply storage/ipfs/api-cli/ipfs-operations.mdx]: Maybe juste  "Understanding IPFS Storage operations" or without ipfs operations

**Pull request closed without merging**